roles/keystone/templates/etc/keystone/keystone.conf

```diff
@@ -109,7 +102,7 @@ remote_id_attribute = {{ sp.remote_id_attribute }}
 
 {% if keystone.federation.enabled|bool  and keystone.federation.sp.saml.enabled|bool -%}
 
-[saml]
+[saml2]
```

Comment:
It seems that when you set the auth methods, Keystone is able to already tell which is the mapped object. And setting an auth method to the mapped object is deprecated. So when we put `saml2`, Keystone can associate that to a config section called `saml2`.